doc/ko/uftrace-graph.md

```diff
@@ -30,6 +30,16 @@ GRAPH 옵션
     사용하여 모든 필드를 숨길 수 있으며 기본 설정은 'total' 이다.
     필드에 대한 상세한 내용은 *FIELDS* 를 참고할 수 있다.
 
+--task
+:   일반적인 함수 그래프 대신 태스크 그래프를 출력한다. 출력된 각 노드들은
+    프로세스 혹은 (초록색으로 표기된)스레드를 보여준다.
+
+\--srcline
+:   가능한 각 함수들의 소스 줄번호를 표시한다.
+
+\--format=*TYPE*
+:   형식화된 출력을 보여준다. 현재로서는, 일반적인 형식과 html 형식이 지원된다.
+
```

Comment:
--task
:   Print task graph instead of normal function graph.  The each node in the
    output shows process or thread(printed in green color).

\--srcline
:   Show source location of each function if available.

\--format=*TYPE*
:   Show format style output. Currently, normal and html styles are supported.